### PR TITLE
Add Function READCVS

### PR DESCRIPTION
**Pull request by davidRFB**
I add a function to read the csv file

### CODE REVIEW

File: Pep_structure.py [hunk 1]
```diff
@@ -11,13 +11,23 @@
               "P":"PRO", "L":"LEU", "F":"PHE", "Y":"TYR", "I":"ILE", "M":"MET", 
               "W":"TRP", "C":"CYS" }
 
-Pep_selection = open("prueba.csv", "r", encoding='utf-8-sig')
-peptides = Pep_selection.readlines()
-list_peptides = []
-for peptide in peptides:
-    list_peptides.append(peptide.strip())
+def ReadCSVPeptide(filename):
```
Comment (dejimenezc) [inline Pep_structure.py:14]: falta documentacion

**Review by dejimenezc — COMMENTED** [review]
Good idea, but lack of documentation

File: Pep_structure.py (hunk 1 above)
Comment (davidRFB) [reply Pep_structure.py:14]: Si !

File: Pep_structure.py [hunk 2]
```diff
@@ -12,6 +12,8 @@
               "W":"TRP", "C":"CYS" }
 
 def ReadCSVPeptide(filename):
+    # inpout filename 
+    # output list with peptides
```
Comment (dejimenezc) [inline Pep_structure.py:16]: super

**Review by dejimenezc — APPROVED** [review]

**Pull request merged**